code/src/containers/forms/product-form/ProductForm.scss

```diff
@@ -0,0 +1,185 @@
+@import "@design-system";
+
+$product-form-language-button-color: #636363;
+
+$gridGap: spacing(5);
+$gridGapHalf: calc($gridGap / 2);
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - $gridGapHalf) calc(70% - $gridGapHalf);
+  grid-gap: $gridGap;
+  margin-bottom: spacing(5);
+
+  &__header {
+    background-color: var(--new-product-form-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+
+    &-main {
+      display: flex;
+      gap: spacing(2);
+      padding: 11.5px spacing(8) !important; // to overwrite mui styles
+      align-items: center;
+    }
+  }
+
+  &__body {
+    padding: spacing(8);
+  }
+
+  &__container {
+    box-shadow: var(--new-product-form-container-shadow, $shadow-black);
+    background-color: var(--new-product-form-container-bg-color, $white);
+  }
+
+  &__main-info-container {
+    flex: 3 0 auto;
+  }
+
+  &__image-preview {
+    border: 1px solid var(--spa-accordion-container-border-color, $gray-light);
+    border-radius: 3px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: spacing(5);
+    aspect-ratio: 1 / 1;
+  }
+
+  &__image-section {
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+  }
+
+  &__main-info-section {
+    grid-column: 2 / 3;
+    grid-row: 1 / 3;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(5);
+  }
+
+  &__additional-info-section {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+
+    .product-form__body {
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__category-select {
+    padding: spacing(3) spacing(11) spacing(3) spacing(14) spacing(3);
```

Comment:
we can use four values for padding isn't it ?

Reply:
Yes, I made a mistake with the fifth value